wifiphisher/common/recon.py

```diff
@@ -168,9 +168,12 @@ def __init__(self, ap_interface, network_manager):
         self._network_manager = network_manager
 
         # filter used to remove non-client addresses
-        self._non_client_addresses = (constants.WIFI_BROADCAST, constants.WIFI_INVALID,
-                                      constants.WIFI_IPV6MCAST1, constants.WIFI_IPV6MCAST2,
-                                      constants.WIFI_SPANNINGTREE, constants.WIFI_MULTICAST)
+        self._non_client_addresses = (constants.WIFI_BROADCAST,
```

Comment:
Without doing a full review but just skimming through it instead of this you can use the `NON_CLIENT_ADDRESSES` in constants which is the same thing.